content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -157,116 +191,293 @@ Automatic reordering rules, enabled by setting the reordering rule's :guilabel:`
+
+- If the forecasted quantity drops below the minimum **on** the forecasted date, replenishment must
+  begin immediately to avoid shortages.
+- If the quantity drops below the minimum **after** the forecasted date, replenishment can wait.
+
+The **To Order** quantity is the total demand on the forecasted date.
+
+By timing purchase orders based on the combined lead times, Odoo optimizes stock levels, keeping
+inventory minimal while ensuring future requirements are ordered at the last possible
+moment—strategic procrastination without the stress!
+
+Common confusion about forecasted quantities
+--------------------------------------------
+
+|SOs| due **after** the :guilabel:`Forecasted Date` are not accounted for in the
+:guilabel:`Forecast` quantities of the reordering rule.
+
+They are, however, accounted for on the forecasted report that is opened by clicking the
+:icon:`fa-area-chart` :guilabel:`(graph)` icon on the replenishment report, as this one represents
+the **long-term forecasted quantity**.
 
 .. example::
-   Referring to the example above, today's date is February twentieth, and the need for the product
-   appears on February twenty-seventh.
 
-   (February 27 - February 20 = 7 days)
+   .. figure:: report/zero-forecast.png
+      :alt: Forecast and To Order quantities is zero.
 
-   Incorrectly setting the :guilabel:`Visibility Days` fewer than seven days in this case results in
-   the need **not** appearing on the replenishment dashboard.
+      Continuing the above example, when the sales order's deadline is adjusted to December 4th, the
+      :guilabel:`Forecast` and :guilabel:`To Order` quantities are zero.
 
-   .. image:: reordering_rules/visibility-days.png
-      :align: center
-      :alt: Show the replenishment dashboard with the correct and incorrect visibility days set.
+   .. figure:: report/five-forecast.png
+      :alt: Show forecasted report.
 
-.. _inventory/product_management/route:
+      Opening the :guilabel:`Forecasted Report` shows the :guilabel:`Forecasted` units is `5.00`.
 
-Preferred route
+.. _inventory/product_management/visibility-days:
+
+Visibility days
 ===============
 
-Odoo allows for multiple routes to be selected under the :guilabel:`Inventory` tab on each product
-form. For instance, it is possible to select both :guilabel:`Buy` and :guilabel:`Manufacture`, thus
-enabling the functionality of both routes.
+*Visibility days* enables the ability to determine if additional quantities should be added to the
+planned replenishment. Odoo checks:
 
-Odoo also enables users to set a preferred route for a product's reordering rule. This is the route
-that the rule defaults to if multiple are selected. To select a preferred route, begin by navigating
-to :menuselection:`Inventory app --> Configuration --> Reordering Rules` or
-:menuselection:`Inventory app --> Operations --> Replenishment`.
+- If forecasted stock on the forecasted date will drop below the minimum in the reordering rule.
```

Comment:
```suggestion
if forecasted stock on the forecasted date will drop below the minimum in the reordering rule.
```

Reply:
I would turn this into a paragraph instead of bullet points, the way things are worded.